lib/scss/components/news-list-item/_news-list-item.scss

```diff
@@ -1,10 +0,0 @@
-@charset "UTF-8";
```

Comment:
This file didn't appear to be included in any compile or scss roll up so I removed it.